.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# review when someone opens a pull request.
+# For more on how to customize the CODEOWNERS file - https://help.github.com/en/articles/about-code-owners
+*       @NuGet/nuget-client @NuGet/nuget-pm
```

Comment:
the nuget.org section of the docs should go to the server team, not the client team

Reply:
I added following clauses in the latest commit. I didn't find @NuGet/nuget-server team like @NuGet/nuget-client. AFAIK the following settings notifies both client and server teams for changes related to either `/docs/nuget-org/` or  `/docs/policies/` directory. Please let me know what you think.

```
# @NuGet/core-team owns any file in the `/docs/nuget-org/` directory
# in the root of your repository and any of its subdirectories.
/docs/nuget-org/ @NuGet/core-team

# @NuGet/core-team owns any file in the `/docs/policies/` directory
# in the root of your repository and any of its subdirectories.
/docs/policies/ @NuGet/core-team
```

I can also make @NuGet/core-team as default reviewers for all the files and then specify the exact locations in the docs that are owned by client team to notify @NuGet/nuget-client team. As per GitHub docs the last team matching the criteria will be notified.